web/app.php

```diff
@@ -27,12 +33,11 @@
 // See http://symfony.com/doc/current/book/http_cache.html#symfony-reverse-proxy
 //
 // require_once __DIR__.'/../app/AppCache.php';
-// $kernel = new AppKernel('prod', false);
-// $kernel->loadClassCache();
 // $kernel = new AppCache($kernel);
 // Request::enableHttpMethodParameterOverride();
```

Comment:
this line should be separate from others IMO. It is not always necessary to uncomment it (it depends of whether you expect this to be enabled or no, and my advice would be to keep it disabled for websites and avoiding to expect a DELETE or PUT request, which would need to be emulated all the time. It simplifies the need to protect against CSRF in places where you would not expect to need this. Websites like this demo application are targetting browsers, so it is fine to take browser limitations into account when designing them IMO)